video_worker/video_images.py

```diff
@@ -146,6 +146,8 @@ def upload(self, generated_images):
             )
             image_keys.append(upload_key.key)
             upload_key.set_contents_from_filename(generated_image)
+            upload_key.set_acl('public-read')
+            upload_key.set_acl('bucket-owner-full-control')
```

Comment:
@yro  I think we should discuss this with Ops. If this is the best way to solve this, then okay from my side.

Reply:
@muzaffaryousaf It's an object owner issue. This was the recommended solution (it's also already in place for video objects)